deeplake/core/dataset/dataset.py

```diff
@@ -3238,6 +3239,34 @@ def _delete_metas(self):
             except KeyError:
                 pass
 
+    def connect(
+        self,
+        org_id: str,
+        token: str,
+        ds_name: Optional[str] = None,
```

Comment:
what would happen if a user calls connect on a DeepLakeCloudDataset? Let's override the connect method in the class and raise an exception there

Reply:
nvm, saw the check below, still would be better to do it in DeepLakeCloudDataset I believe

Reply:
Added the overridden method in DeepLakeCloudDataset